MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md

```diff
@@ -0,0 +1,35 @@
+# MIP4c2-SP22: MIP41 Facilitator Offboarding Amendment
+
+## Preamble
+
+```
+MIP4c2-SP#: 22
+MIP to be amended: MIP41
+Author(s): Sam MacPherson (@hexonaut)
+Contributors:
+Tags: MIP41, facilitator
+Status: Withdrawn
+Date Proposed: 2022-07-03
+Date Ratified: N/A
+Forum URL: https://forum.makerdao.com/t/mip4c2-sp22-mip41-facilitator-offboarding-amendment/16297
+```
```

Comment:
Can you add an Extra field here?: "This proposal has been [withdrawn by its author](https://forum.makerdao.com/t/mip4c2-sp22-mip41-facilitator-offboarding-amendment/16297/36)" or something along those lines.

Reply:
I think adding a link to the public post is a good idea: `[withdrawn by its author](https://forum.makerdao.com/t/mip4c2-sp22-mip41-facilitator-offboarding-amendment/16297/36)`

Reply:
done